frontend/src/app/view/view.component.ts

```diff
@@ -123,25 +142,21 @@ export class ViewComponent {
     this.showConfirmDialog = false;
     this.deleteId = '';
   }
+
   delete() {
     const id = this.deleteId;
     if (!!id) {
-      this.http.delete(`/api/files/${id}`).subscribe(res => {
-        this.showConfirmDialog = false;
-        this.deleteId = '';
-        // const newFileArr = this.listArr;
-        // const index = newFileArr.findIndex((file) => file.id === id);
-        // if (index >= 0) {
-          // newFileArr.splice(index,1);
-          // this.fileSubject.next([...newFileArr]);
-        // }
-        this.router.navigate(['list/']);
-      },
-      err => {
-        if (err.status === 404) {
-          alert('The file you are trying to upload/update does not exist. Please update/upload a correct file.');
+      this.http.delete(`/api/files/${id}`).subscribe(
+        res => {
+          this.showConfirmDialog = false;
+          this.deleteId = '';
+          this.router.navigate(['list/']);
+        },
+        err => {
+          if (err.status === 404) {
+            alert('The file you are trying to upload/update does not exist. Please update/upload a correct file.');
```

Comment:
issue(blocking): Avoid using `alert()`. Let's show an Angular modal dialog instead.

Reply:
fixed in latest commit [#2187b4](https://github.com/GoogleCloudPlatform/app-large-data-sharing-golang/pull/1/commits/2187b4e6fd3f554fd1975d3a0a9e7991e8da0a5a)